app/helpers/federation_helper.rb

```diff
@@ -234,8 +238,9 @@ def counts_ontology_ids_by_portal_name(portals_ids)
       counts[current_portal.downcase] += 1 if id.include?(current_portal.to_s.downcase)
 
       federation_portals.each do |portal|
-        portal_api = federated_portals[portal.downcase.to_sym][:api]
-        counts[portal.downcase] += 1 if id.include?(portal_api)
+        portal_api = federated_portals[portal.downcase.to_sym][:api].sub(/^https?:\/\//, '')
```

Comment:
@syphax-bouazzouni I added `sub(/^https?:\/\//, '')` here to make sure to deal with http, https variations.
cause I found for example the ontology STY in earthportal is using http `"http://earthportal.eu/ontologies/STY"` while the UI link is using https `"https://ecoportal.lifewatch.eu/"`

Reply:
Yeah, sometimes you can even get, an IP, not the actual domain name. but anyway for now good enough.